NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -190,7 +191,8 @@ def run(
             event,
             station,
             detector,
-            passband=None
+            passband=None,
+            excluded_channel_ids=None
```

Comment:
Can we use `excluded_channels` (also in the docstring), just because that's what the genericNoiseAdder does as well?